### PR TITLE
fix typo for namespace in codeflare

### PR DESCRIPTION
**Pull request by zdtsw**
ref: https://issues.redhat.com/browse/RHODS-12812

### CODE REVIEW

File: controllers/dscinitialization/utils.go [hunk 1]
```diff
@@ -238,6 +238,17 @@ func (r *DSCInitializationReconciler) reconcileDefaultNetworkPolicy(ctx context.
 							},
 						},
 					},
+					{ // OR logic for PSI
+						From: []netv1.NetworkPolicyPeer{
+							{ // need this for access dashboard
```
Comment (anishasthana) [inline controllers/dscinitialization/utils.go:243]:
```suggestion
							{ // need this to access dashboard
```

File: controllers/dscinitialization/utils.go [hunk 2]
```diff
@@ -238,6 +238,17 @@ func (r *DSCInitializationReconciler) reconcileDefaultNetworkPolicy(ctx context.
 							},
 						},
 					},
+					{ // OR logic for PSI
```
Comment (anishasthana) [inline controllers/dscinitialization/utils.go:241]: I'm curious about the need for this PSI logic block.

**Review by anishasthana — COMMENTED** [review]
The same PR will also need to go to main right?

**Comment by zdtsw** [comment]
> The same PR will also need to go to main right?

yes, i will need to backport to main next week.

**Comment by zdtsw** [comment]
merge it based on the offline conversation.

**Comment by zdtsw** [comment]
Screenshot from 2023-11-09 08-57-02
post screenshot from my local tests after with this fix

**Pull request merged**